library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt

```diff
@@ -29,9 +29,11 @@ internal object FormatUtils {
 
     fun formatHeaders(httpHeaders: List<HttpHeader>?, withMarkup: Boolean): String {
         return httpHeaders?.joinToString(separator = "") { header ->
-            (if (withMarkup) "<b>" else "") + header.name + ": " +
-                (if (withMarkup) "</b>" else "") + header.value +
-                if (withMarkup) "<br />" else "\n"
+            if (withMarkup) {
+                "<b>" + header.name + ": " + "</b>" + header.value + "<br />"
+            } else {
+                header.name + ": " + header.value + "\n"
```

Comment:
Can you use when applicable the "${header.name}: ${header.value}\n" way of doing strings? Thanks